eclair-core/src/test/scala/fr/acinq/eclair/channel/states/StateTestsHelperMethods.scala

```diff
@@ -86,7 +88,9 @@ trait StateTestsHelperMethods extends TestKitBase {
                           relayerA: TestProbe,
                           relayerB: TestProbe,
                           channelUpdateListener: TestProbe,
-                          wallet: EclairWallet) {
+                          wallet: EclairWallet,
+                          alicePeer: TestProbe,
+                          bobPeer: TestProbe) {
```

Comment:
Nice addition, it will be useful to test that we disconnect appropriately in some cases.